hack/make/dyngccgo

```diff
@@ -15,7 +16,7 @@ fi
 
 (
 	export IAMSTATIC="false"
-	export EXTLDFLAGS_STATIC_DOCKER=''
+	export EXTLDFLAGS_STATIC_DOCKER='-static-libgcc -static-libgo -lnetgo'
```

Comment:
since your other patch chunk adds `-lnetgo` to always be passed in `hack/make/gccgo`, it doesn't need to be part of `EXTLDFLAGS_STATIC_DOCKER` or am I missing something?